src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/ClaimMotionStakes/ClaimMotionStakes.tsx

```diff
@@ -4,23 +4,32 @@ import DetailItem from '~shared/DetailsWidget/DetailItem';
 import { MotionData } from '~types';
 import { useClaimWidgetConfig } from '~hooks';
 
+import styles from './ClaimMotionStakes.css';
+
 const displayName = `common.ColonyActions.ActionDetailsPage.DefaultMotion.ClaimMotionStakes`;
 
 interface ClaimMotionStakesProps {
   motionData: MotionData;
   startPollingAction: (pollInterval: number) => void;
 }
 
+export type ClaimMotionStakesStyles = typeof styles;
+
 const ClaimMotionStakes = ({
   motionData,
   startPollingAction,
 }: ClaimMotionStakesProps) => {
-  const config = useClaimWidgetConfig(motionData, startPollingAction);
+  const config = useClaimWidgetConfig(motionData, startPollingAction, styles);
```

Comment:
Correct me if I'm wrong, but do they have to go through the hook? Looks like there's no logic involved in deciding which ones to apply and the `claimLabel` class could as well be passed to `DetailItem` directly

Reply:
I only want to put the style on the first item (the claim detail), whereas applying it to the DetailItem will apply it to all of them